prometheus-to-sd/translator/stackdriver.go

```diff
@@ -53,15 +54,21 @@ func SendToStackdriver(service *v3.Service, config *config.CommonConfig, ts []*v
 			defer wg.Done()
 			req := &v3.CreateTimeSeriesRequest{TimeSeries: ts[begin:end]}
 			_, err := service.Projects.TimeSeries.Create(proj, req).Do()
+			now := time.Now()
```

Comment:
What are you expecting to measure when error occurs?
E.g. SD is down, 500 will be returned imminently. This can hide potential problem with latency of successful requests.

Reply:
It makes sense to measure latency only for delivered timeseries, for failed that's obviously infinite. Haven't noticed that functions is not returning after error.